src/server/supplements/har/har.ts

```diff
@@ -81,6 +81,7 @@ export type Response = {
   redirectURL: string;
   headersSize: number;
   bodySize: number;
+  _transferSize: number;
```

Comment:
why not name it `transferSize` right away? Is it because the field is non-standard?

Reply:
Exactly, its a chromium only field and not yet part of the official spec.